scripts/release_scripts/cut_release_or_hotfix_branch.py

```diff
@@ -318,13 +318,8 @@ def execute_branch_cut(target_version, hotfix_number):
             'branch protection rule and doing all the cherrypicks.')
 
     common.ask_user_to_confirm(
-        'Ask Sean (or Ben, if Sean isn\'t available) to create '
-        'a new branch protection rule by:\n'
-        '1. Going to this page: https://github.com/oppia/oppia/'
-        'settings/branch_protection_rules/new.\n'
-        '2. Typing in the full branch name %s.\n'
-        '3. Checking the box: Restrict who can push to matching '
-        'branches (then add the oppia/release-coordinators team)\n' % (
+        'Confirm with Sean (or Ben, if Sean isn\'t available) that '
```

Comment:
Thanks, @ankita240796! You can actually drop this whole thing though. I managed to figure out a way to use regexes on github so that the one rule matches all the branches going forward :)

Reply:
Created a new one, I made the branch cut incorrectly from oppia repo instead of my fork.